cypress/e2e/Transcript/Access/ThereAreNoMissingBoardDetails.feature

```diff
@@ -0,0 +1,10 @@
+Feature: The user can access the transcripts
+  Background:
+    Given I am a registered user
+    And I provided the required <MedicalBoardInformation> if any
+
+  Scenario: user has no missing board details
```

Comment:
We should mention what happens with the transcript access (I know is in the feature)

Reply:
which line you mean? can you give an example please

Reply:
Line 6, the scenario title.